pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -1,3 +1,13 @@
 package preflight
 
 // Add windows specific checks and fixes
+
+const (
+	DefaultVMDriver = "virtualbox"
+)
+
+var (
```

Comment:
👍

Reply:
This would be wrong, as defaults can not be changed afterwards.

It would be better to have hyper-v as the default, but print as error that this is not supported yet.